### DISCUSSION TITLE
Error on Deleting record (local.ERROR: Missing required parameter /{record}/edit] [Missing parameter: record])

### DISCUSSION THREAD

**Discussion by pedrocoelho98** [filamentphp/filament#3972] [Help] [answered]
I'm getting this error when deleting on record from the form. It pops-up but then goes away and the delete action is done.

`local.ERROR: Missing required parameter for [Route: filament.resources.reservations.edit] [URI: admin/reservations/{record}/edit] [Missing parameter: record]`

The record gets deleted, but along the way the error debug page briefly shows.

Any thoughts?

**Comment by danharrin** [comment, marked as answer]
Are you on the latest version? I think I fixed something like this a few days ago

**Reply by pedrocoelho98** [reply]
Yep. That solved it!

**Comment by gotzsys** [comment]
I have the same issue inside the edit page, when I click on the delete button.
I'm running the latest version:
"filament/filament": "^2.16",

[2022-12-09 21:37:01] local.ERROR: Missing required parameter for [Route: filament.resources.environments.edit] [URI: environments/{record}/edit] [Missing parameter: record]. {"userId":1,"exception":"[object] (Illuminate\\Routing\\Exceptions\\UrlGenerationException(code: 0): Missing required parameter for [Route: filament.resources.environments.edit] [URI: environments/{record}/edit] [Missing parameter: record]. at /var/www/html/vendor/laravel/framework/src/Illuminate/Routing/Exceptions/UrlGenerationException.php:35)
[stacktrace]

![tela1](https://user-images.githubusercontent.com/13874850/206819476-be78191d-49ac-4cd2-ab58-d978ea6c4811.jpg)

**Reply by danharrin** [reply]
Please open an issue with a reproduction repo

**Reply by altimusin** [reply]
Hi All,

Here I tried to create & reproduces the error.

https://github.com/altimusin/error_on_deleting.git

        "php": "^8.0.2",
        "filament/filament": "^2.16",
        "guzzlehttp/guzzle": "^7.2",
        "laravel/framework": "^9.19",
        "laravel/sanctum": "^3.0",
        "laravel/tinker": "^2.7"


Here my screen shoot 
- before delete button pressed.
![image](https://user-images.githubusercontent.com/43429317/207515646-ec9b4558-96a8-4f5b-9e1f-6abfd4faf461.png)

- after delete button pressed.
![image](https://user-images.githubusercontent.com/43429317/207515981-e227faff-71f1-435c-9f80-6b661a64ad86.png)


composer install
create .env file & db connection
php artisan migrate:fresh --seed
php artisan serve
user : [admin@admin.com](mailto:admin@admin.com) password : password

**Reply by danharrin** [reply]
#5240
